.build/parent-pom-template.xml

```diff
@@ -705,7 +756,7 @@
       <dependency>
         <groupId>net.openhft</groupId>
         <artifactId>chronicle-queue</artifactId>
-        <version>5.20.123</version>
```

Comment:
no bom, as requested we are upgrading to the latest versions (that are available in the maven1 repo)

Reply:
And those latest versions still correspond to specific BOM number, probably latest and I see in the past we were adding those looking at a BOM as my understanding is that the versions of the different dependencies which are listed in a BOM have been integrated and tested to work together.
I was looking into this one - https://mvnrepository.com/artifact/net.openhft/chronicle-bom/2.23.137
Then I also realized that we do not use the latest ea release (as posix) but the previous one
https://mvnrepository.com/artifact/net.openhft/chronicle-queue

At first I thought you do not want to use ea version, but then you used for posix?

Reply:
I started off with the bom, then going through maven1 repo used the versions available (basically aligning to the bom, some patch versions differed).

Then fixing tests some versions needed bumping, posix the example here.

Then on the dev@ thread folk requested we use the latest versions available. The versions that then got bumped made sense to as they matched sibling libraries we already had (e.g. jna*), and/or were about newer patch versions supporting more arch/platforms.

While the initial goal may have been to re-use the tested QA of the bom's declaration, that's evolved to tailoring it best (with evidence) to our needs and relying on being in trunk to do our own testing and QA.  (IMO this approach would not be appropriate in one of our release branches.)

Reply:
I am still not sure why posix is at ea latest version but the chronicle-* dependencies are at non-ea  latest versions.
Also, maybe a typo but chronicle-core has 2.23.37, you have 36 here.

Reply:
> I am still not sure why posix is at ea latest version but the chronicle-* dependencies are at non-ea latest versions.

Only the bump of posix was needed to fix the related failures.

( Many, and newer, versions are simply not found in https://repo1.maven.org/maven2/net/openhft/ )

> Also, maybe a typo but chronicle-core has 2.23.37, you have 36 here.

While chronicle-core-2.23.37 itself is available publicly, it's parent pom is not, and uses other dependencies that are SNAPSHOTs. So that version is out for us.

Reply:
Thank you for explaining, I see what you mean now! 
Can we add a few sentences to the ticket about that? I know people who are rebasing their forks are doing another pass of review so it might be helpful for context as it is not immediately obvious.